scripts/build/builders/host.py

```diff
@@ -36,6 +36,9 @@ class HostApp(Enum):
     OTA_REQUESTOR = auto()
     PYTHON_BINDINGS = auto()
     NL_TEST_RUNNER = auto()
+    LIGHTING = auto()
+    TV_CASTING_APP = auto()
```

Comment:
remove `_APP` suffix here as well. Apparently TV_APP is the one-off above, maybe looks ok because the name is very short, however prefferably I would remove this as well and just use `TV`

Reply:
OK done